spec/models/miq_report_result_spec.rb

```diff
@@ -1,14 +1,62 @@
 describe MiqReportResult do
-  context "persisting generated report results" do
-    before(:each) do
-      EvmSpecHelper.local_miq_server
+  before :each do
+    EvmSpecHelper.local_miq_server
+
+    @user1 = FactoryGirl.create(:user_with_group)
+  end
+
+  context "report result created by User 1 with current group 1" do
```

Comment:
Maybe something telling us what this context is for instead of what it does?

Maybe something like `report result filtering` ?